src/Host/Client/Impl/Session/RSessionEvaluationCommands.cs

```diff
@@ -222,11 +222,13 @@ public static Task OptionsSetWidthAsync(this IRExpressionEvaluator evaluation, i
 
         public static async Task SetCodePageAsync(this IRExpressionEvaluator evaluation, int codePage, CancellationToken cancellationToken = default(CancellationToken)) {
             var cp = $".{codePage}";
-            if (codePage == 0) {
-                cp = await evaluation.IsRSessionPlatformWindowsAsync() ? ".437" : "en_US.UTF-8";
+            if (codePage == 0 && !await evaluation.IsRSessionPlatformWindowsAsync(cancellationToken)) {
+                cp = "en_US.UTF-8";
```

Comment:
I'm confused about how this works. So now if this function gets called with codePage==0, we try to set LC_CTYPE to `".0"` - but that's not a valid codepage, is it?